content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md

```diff
@@ -188,11 +188,33 @@ openssl command is expected.
     $ rm -rf certs
     {{< /text >}}
 
-*   Remove the secret `cacerts`, and the `foo` and `istio-system` namespaces:
+*   Remove the secret `cacerts`:
 
     {{< text bash >}}
     $ kubectl delete secret cacerts -n istio-system
-    $ kubectl delete ns foo istio-system
     {{< /text >}}
 
-*   To remove the Istio components: follow the [uninstall instructions](/docs/setup/getting-started/#uninstall) to remove.
+*   Remove the authentication policy from the `foo` namespace:
+
+    {{< text bash >}}
+    $ kubectl delete peerauthentication -n foo default
+    {{< /text >}}
+
+*   Remove the sample applications `sleep` and `httpbin`:
+
```

Comment:
You need to run the `make snips` command to generate the snips.sh correctly. This still has all these commands that are not needed.

Reply:
This does not make sense. I thought the snips and test.sh in this PR was for this task. Looks like you are pulling in changes from your old PR. This needs to be cleaned up.